python_ta/__init__.py

```diff
@@ -63,6 +63,7 @@ def check_errors(
     output: Optional[Union[str, IO]] = None,
     load_default_config: bool = True,
     autoformat: Optional[bool] = False,
+    on_verify_fail: str = "log",
```

Comment:
A more appropriate type annotation is `Literal`(https://docs.python.org/3/library/typing.html#typing.Literal)